cachetory/serializers/json.py

```diff
@@ -0,0 +1,19 @@
+from __future__ import annotations
+
+import json
+from typing import Generic
+
+from cachetory.interfaces.serializers import Serializer, ValueT
+
+
+class JsonSerializer(Serializer[ValueT, bytes], Generic[ValueT]):
```

Comment:
It should be `str`, not `bytes` (`json.dumps()` returns `str`)

Reply:
updated